src/test/kotlin/com/igorwojda/integer/generateallpairs/challenge.kt

```diff
@@ -4,7 +4,15 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun getAllPairs(n: Int): List<Pair<Int, Int>> {
```

Comment:
Complex challenge: ai abordat iterativ foarte bine. Poti completa implementarea urmatoare care foloseste doar transformari? (Nu bag mana in foc ca nu trebuie sa schimbi operatori :D )
```
return (0..n).map { ... }.flatMapIndexed { ... } 
```